examples/AspNetCore/Controllers/WeatherForecastController.cs

```diff
@@ -54,11 +60,25 @@ public IEnumerable<WeatherForecast> Get()
         })
         .ToArray();
 
+        // Log using ILogger API.
         this.logger.LogInformation(
             "WeatherForecasts generated {count}: {forecasts}",
             forecast.Length,
             forecast);
 
+        // Log using LogEmitter API.
+        this.logEmitter.Log(
```

Comment:
I wonder if it makes sense to demonstrate this here? My understanding is that this isn't really meant to be used by end users, but by authors of extensions for various logging libraries like serilog or log4net. Have you had a chance to experiment with the API in the context of some other log framework?